extensions/short-io/README.md

```diff
@@ -0,0 +1,15 @@
+# Short.io
+
+Shorten links using your own domain via Short.io.
+
+**Tips**:
+
+- The first time you use 𝐒𝐡𝐨𝐫𝐭𝐞𝐧 𝐋𝐢𝐧𝐤 and 𝐒𝐞𝐚𝐫𝐜𝐡 𝐋𝐢𝐧𝐤𝐬 commands, they will use the first domain of your domains list.
+- You can set default domain for 𝐒𝐡𝐨𝐫𝐭𝐞𝐧 𝐋𝐢𝐧𝐤 and 𝐒𝐞𝐚𝐫𝐜𝐡 𝐋𝐢𝐧𝐤𝐬 commands via the 𝐒𝐡𝐨𝐫𝐭𝐞𝐧 𝐋𝐢𝐧𝐤 𝐖𝐢𝐭𝐡 𝐃𝐨𝐦𝐚𝐢𝐧 command (action: Set Default Domain, shortcut: Cmd+Enter).
```

Comment:
Sorry for nitpicking, but could you keep all in the same font? I see the commands in a different one like: 𝐒𝐞𝐚𝐫𝐜𝐡 𝐋𝐢𝐧𝐤𝐬. If you want to highlight you can use bold or code instead

Reply:
Thanks for the suggestion, I will use Blod to bold them.